black.py

```diff
@@ -2357,6 +2376,44 @@ def is_python36(node: Node) -> bool:
     return False
 
 
+def get_future_imports(node: Node) -> Set[str]:
```

Comment:
I find that the number of isinstance checks here makes the function harder to read than necessary. I wonder if we can do anything to minimize those checks.

Reply:
Mypy made me add some but I think I overdid it. Your other comments helped me find a few unnecessary ones, so I think it's better now.